src/components/forms/pool_actions/AddLiquidityForm/AddLiquidityForm.vue

```diff
@@ -101,14 +100,13 @@ const joinTokensWithoutBalance = computed<string[]>(() =>
 );
 
 async function initializeTokensForm(isSingleAssetJoin: boolean) {
-  setAmountsIn([]);
```

Comment:
I think this might have caused it, but can't see how that would also get past the existing guard in the query function.